src/pages/developers/learning-tools.tsx

```diff
@@ -122,7 +122,7 @@ export const getStaticProps = (async ({ locale }) => {
     "/developers/learning-tools"
   )
 
-  const contentNotTranslated = !existsNamespace(locale!, requiredNamespaces[2])
+  const contentNotTranslated = !existsNamespace(locale!, requiredNamespaces[1])
```

Comment:
`1` is always going to be the ns we are looking for?

Reply:
Just reviewing this change myself... To answer your question @pettinarip, yes, for every page _except_:
- src/pages/404.tsx
- src/pages/index.tsx

These pages are the only ones that simply look at "common", which should always be available for any language we list. We're currently looking up `requiredNamespaces[0]` for those two pages, but we could arguably just simply to `const contentNotTranslated = false` for the homepage and 404 page, since we know `common.json` will always be available.

---

Separately, we discussed previously the option to abstract the `getStaticProps` logic for declaring `requiredNamespaces`, `contentNotTranslated` and `lastDeployDate` into a single helper function to minimize some of this redundancy... I think the above changes would help with this, as we could assume `requiredNamespace[1]` for all pages, and just pass a boolean for the home/404 page to override (or some variant of this).